conda-store-server/pyproject.toml

```diff
@@ -25,55 +29,77 @@ classifiers = [
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3 :: Only",
 ]
+
+dynamic = ["version"]
+
 dependencies = [
+  # env builds
+  # conda (which cannot be installed via pip see https://github.com/conda/conda/issues/11715)
+  "python-docker",
+  # we need platform dependent dependencies here
   "conda-docker; sys_platform == 'linux'",
+  "conda-lock >=1.0.5",
   "conda-pack",
+  "conda-package-handling",
+  "conda-package-streaming",
+  # web server
+  "alembic",
   "celery",
-  "sqlalchemy",
-  "requests",
   "fastapi",
+  "filelock",
+  "itsdangerous",
+  "jinja2",
+  "pyjwt",
+  "psycopg2-binary",
+  "pymysql",
   "pyyaml",
   "redis",
+  "requests",
   "pydantic<2",
-  "minio",
+  "python-multipart",
+  "sqlalchemy<=1.4.47",
   "traitlets",
-  "pyjwt",
+  "uvicorn",
   "yarl",
-  "filelock",
-  "itsdangerous",
-  "jinja2",
-  "python-multipart",
-  "python-docker"
-  # conda (which should not be included here)
+  # installer
+  "constructor",
```

Comment:
constructor is a runtime dependency and needs to be in the environment for it to work, which is why you're seeing these unit test failures:

```
tests/test_actions.py::test_generate_constructor_installer
```

Reply:
Exactly is a runtime dependency not a dev dependency. So it does not need to be in the environment.